cfgov/v1/atomic_elements/organisms.py

```diff
@@ -489,7 +489,7 @@ def __init__(self, table_options=None, attrs=None):
     def render(self, name, value, attrs=None):
         value = self.json_dict_apply(
             value,
-            partial(expand_db_html, for_editor=True)
+            expand_db_html
```

Comment:
You can revert this change; it'll be included in my PR for the `TableBlock` Hallo stuff.

```suggestion
            partial(expand_db_html, for_editor=True)
```

Reply:
I dropped the commit that made that change.